src/app/static/src/app/components/header/HintrVersionMenu.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <drop-down :text="`v${hintrVersions.naomi}`" :right="true" style="flex: none">
+    <a class="dropdown-item" href="#"> naomi: v{{ hintrVersions.naomi }} </a>
```

Comment:
These shouldn't actually be links! Maybe `<span class="dropdown-item">` with `style=cursor: default` to stop the cursor being styled like a link cursor.